nih-sparc/sparc-app
# Dataset Detail Page - Initial Setup and Content #4a74qk #4a9fy9 #4a9g17

# Description

The purpose of this PR is to setup and wire the Dataset Detail Page. The following on the page has been completely wired to the endpoint:

1. Header
2. Description Tab
3. Files Tab

The following are still incomplete pending a few items:

1. About tab needs to have the project ID/sparc award associated with it
2. Linkable path in the files tab needs to point to `packages` instead of `Root`. This is dependent on a minor fix that Bo is working on.

The metadata tab has also been removed as it's no longer needed.

## Type of change


# How Has This Been Tested?

1. Navigate to the Find Data page and click on the Datasets tab
2. Click on a dataset in the table. The details page should be displayed and function as expected.

# Checklist:
